health-services/project-factory/src/server/utils/genericUtils.ts

```
@@ -1303,12 +1307,12 @@
     mappedRowData[boundaryCodeIndex] = boundaryCode;
     return mappedRowData;
   });
-  if(type == "boundaryManagement"){
+  if (type == "boundaryManagement") {
     logger.info("Processing data for boundaryManagement type")
     const latLongBoundaryMap = await getLatLongMapForBoundaryCodes(request, boundaryCodeList);
     for (let d of data) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'd' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)